### DISCUSSION TITLE
Missing repo data from the Github APIs

### DISCUSSION THREAD

**Discussion by av06** [community/community#53728] [Apps, API and Webhooks]
### Select Topic Area

Bug

### Body

Hello Github Team,

I am Akhil, the Co-founder at Scoutflo (https://scoutflo.com) We’re building a marketplace for commercial open source companies.
Right now we’re using a combination of manual data being fed + Using Github API’s to populate product information and the relevant repo’s for those products via our internal CMS tool.

We’ve seen a few cases where we are not able to get the data of certain repos.

A few product repos include:

https://github.com/kong/kong
https://github.com/SigNoz/signoz
https://github.com/frappe/erpnext
https://github.com/gradle
https://github.com/OpenNebula
https://github.com/getsentry/

Example:
https://api.github.com/users/SigNoz/repos not returning the signoz/signoz repo
Do let us know what’s the possible issue and a fix for this.

Regards,
Akhil

**Comment by ghostinhershell** [comment]
Hey @av06, thanks so much for reaching out here and welcome to the Community on Discussion! Can you tell us a but more about how you're making these API calls? Also, can you provide us an x-github-request-id for the requests so we can see how they are failing? Examples of error messages and attempts from your side would help us dig into this issue further. 

Thanks!

**Reply by tadija** [reply]
hey @ghostinhershell i have the same issue as reported here, it might have something to do with repositories that were renamed (in my case those are missing)

**Comment by 75lb** [comment]
I'm having the same issue - missing repos.. I'm running this query:

```graphql
query getUserRepoNames ($login: String!, $after: String, $privacy: RepositoryPrivacy) {
  user (login: $login) {
    repositories (first: 100, after: $after, ownerAffiliations: [OWNER, COLLABORATOR, ORGANIZATION_MEMBER], privacy: $privacy) {
      totalCount
      pageInfo {
        endCursor
        hasNextPage
      }
      edges {
        node {
          nameWithOwner
        }
      }
    }
  }
}
```

with these variables:

```json
{
  "login": "75lb",
  "privacy": "PRIVATE"
}
```

which should return all the private repos I own, across all organisations.. however, many are missing.. for example, I own the `client-zone` organisation which has 10 private repos - however, the above query only returns 5 of those repos. All 10 repos are correctly returned from the v3 REST API, as a result I try to avoid the graphQL API where possible.
